src/components/FlawForm.vue

```diff
@@ -466,6 +465,7 @@ const createdDate = computed(() => {
             @loadInternalComments="loadInternalComments"
           />
         </div>
+        <FlawHistory :flaw="flaw" @expandFocusedComponent="expandFocusedComponent" />
```

Comment:
Maybe we should add a check like we do with the comments so this section is hidden when creating a flaw?
```suggestion
        <FlawHistory v-if="mode === 'edit'" :flaw="flaw" @expandFocusedComponent="expandFocusedComponent" />
```

Reply:
After further discussion, we decided to leave as it

Reply:
Off thread discussion output:
Makes sense to use conditional render checks inside the component when the check data is available.
In the comments section the data for the render check comes from a previous render request, that's the reason the render condition is specified in the parent (edit mode).